frontends/p4/tableKeyNames.cpp

```diff
@@ -76,7 +76,7 @@ class KeyNameGenerator : public Inspector {
 
     cstring getName(const IR::Expression* expression) {
         if (name.find(expression) == name.end())
-            return "";
+            return cstring();
```

Comment:
This check is unneded -- `::get` will return `cstring()` if the expression is not in the map.